aten/src/ATen/native/LossMultiMargin.cpp

```diff
@@ -262,12 +266,9 @@ Tensor multi_margin_loss_cpu(
     const Tensor& input,
     const Tensor& target,
     const Scalar& p,
-    const Scalar& margin, const c10::optional<Tensor>& weight_opt,
+    const Scalar& margin,
+    const c10::optional<Tensor>& weight,
     int64_t reduction) {
-  // See [Note: hacky wrapper removal for optional tensor]
-  c10::MaybeOwned<Tensor> weight_maybe_owned = at::borrow_from_optional_tensor(weight_opt);
-  const Tensor& weight = *weight_maybe_owned;
-
```

Comment:
Here and elsewhere: to have a consistent API for `multi_margin_loss_shape_check` (it accepts optional `weight` on CPU and CUDA).